lightning/src/ln/channelmanager.rs

```diff
@@ -3379,127 +3380,36 @@ where
-							log_trace!(self.logger, "Sending funding_locked and announcement_signatures for {}", log_bytes!(channel.channel_id()));
-							pending_msg_events.push(events::MessageSendEvent::SendAnnouncementSignatures {
-								node_id: channel.get_counterparty_node_id(),
-								msg: announcement_sigs,
-							});
-						} else {
-							log_trace!(self.logger, "Sending funding_locked WITHOUT announcement_signatures for {}", log_bytes!(channel.channel_id()));
-						}
-						short_to_id.insert(channel.get_short_channel_id().unwrap(), channel.channel_id());
-					}
-				} else if let Err(e) = res {
-					if let Some(short_id) = channel.get_short_channel_id() {
-						short_to_id.remove(&short_id);
-					}
-					// It looks like our counterparty went on-chain or funding transaction was
-					// reorged out of the main chain. Close the channel.
-					failed_channels.push(channel.force_shutdown(true));
-					if let Ok(update) = self.get_channel_update(&channel) {
-						pending_msg_events.push(events::MessageSendEvent::BroadcastChannelUpdate {
-							msg: update
-						});
-					}
-					pending_msg_events.push(events::MessageSendEvent::HandleError {
-						node_id: channel.get_counterparty_node_id(),
-						action: msgs::ErrorAction::SendErrorMessage { msg: e },
-					});
-					return false;
-				}
-				true
-			});
-
-			if let Some(height) = height_opt {
-				channel_state.claimable_htlcs.retain(|&(ref payment_hash, _), htlcs| {
-					htlcs.retain(|htlc| {
-						// If height is approaching the number of blocks we think it takes us to get
-						// our commitment transaction confirmed before the HTLC expires, plus the
-						// number of blocks we generally consider it to take to do a commitment update,
-						// just give up on it and fail the HTLC.
-						if height >= htlc.cltv_expiry - HTLC_FAIL_BACK_BUFFER {
-							let mut htlc_msat_height_data = byte_utils::be64_to_array(htlc.value).to_vec();
-							htlc_msat_height_data.extend_from_slice(&byte_utils::be32_to_array(height));
-							timed_out_htlcs.push((HTLCSource::PreviousHopData(htlc.prev_hop.clone()), payment_hash.clone(), HTLCFailReason::Reason {
-								failure_code: 0x4000 | 15,
-								data: htlc_msat_height_data
-							}));
-							false
-						} else { true }
-					});
-					!htlcs.is_empty() // Only retain this entry if htlcs has at least one entry.
-				});
-			}
-		}
-
-		self.handle_init_event_channel_failures(failed_channels);
-
-		for (source, payment_hash, reason) in timed_out_htlcs.drain(..) {
-			self.fail_htlc_backwards_internal(self.channel_state.lock().unwrap(), source, &payment_hash, reason);
-		}
-	}
-
 	/// Updates channel state to take note of transactions which were confirmed in the given block
 	/// at the given height.
 	///
-	/// Note that you must still call (or have called) [`update_best_block`] with the block
+	/// Note that you must still call (or have called) [`best_block_updated`] with the block
```

Comment:
I think we should basically drop the docs here - users should refer to the docs on `chain::Confirm`.

Reply:
Done throughout and in `ChannelMonitor`. Left a reference in the latter since it doesn't directly implement `Confirm`.